lib/install/management/appliance.go

```diff
@@ -490,21 +485,31 @@ func (d *Dispatcher) createAppliance(conf *config.VirtualContainerHostConfigSpec
 		return err
 	}
 
-	var info *types.TaskInfo
-	// create appliance VM
-	if d.isVC && d.vchVapp != nil {
-		info, err = tasks.WaitForResult(d.op, func(ctx context.Context) (tasks.Task, error) {
-			return d.vchVapp.CreateChildVM(ctx, *spec, d.session.Host)
-		})
-	} else {
-		// if vapp is not created, fall back to create VM under default resource pool
-		info, err = tasks.WaitForResult(d.op, func(ctx context.Context) (tasks.Task, error) {
-			return d.session.VMFolder.CreateVM(ctx, *spec, d.vchPool, d.session.Host)
-		})
+	// Create the VCH inventory folder
+	if d.isVC {
+		folderPath := fmt.Sprintf("%s", path.Join(d.session.VMFolder.InventoryPath, conf.Name))
```

Comment:
Why do you need fmt.Sprintf here? seems like not necessary at all.

Reply:
I _think_ that was from a previous tenant...anyhoo...I've removed it...provides little to no value...